cobra/__init__.py

```diff
@@ -1158,17 +1160,52 @@ def initSocketBuilder(host,port):
         socket_builders[ (host,port) ] = builder
     return builder
 
-def startCobraServer(host="", port=COBRA_PORT):
+def startCobraServer(host="", port=COBRA_PORT, sslca=None, sslcrt=None, sslkey=None, msgpack=True):
+    # TODO: replace this with getCobraDaemon code?
     global daemon
     if daemon is None:
-        daemon = CobraDaemon(host,port)
+        daemon = CobraDaemon(host, port, sslca=sslca, sslcrt=sslcrt, sslkey=sslkey, msgpack=msgpack)
         daemon.fireThread()
     return daemon
 
-def runCobraServer(host='', port=COBRA_PORT):
-    daemon = CobraDaemon(host,port)
+daemon_threads = {}
```

Comment:
If we're going to keep this around, can we move this to the top of the file so this isn't a pain to find?

Reply:
sure.